perception/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -303,6 +318,7 @@ bool OccupancyMapMonitor::loadMapCallback(moveit_msgs::LoadMap::Request& request
   tree_->lockWrite();
   try
   {
+    tree_->enableChangeDetection(false);
```

Comment:
Is there any good reason to explicitly disable change detection here? This seems unrelated to the PR.